src/utils/aquarius.ts

```diff
@@ -97,3 +99,47 @@ export async function getAssetsNames(
     }
   }
 }
+
+export async function getAlgorithmsOptions(
```

Comment:
nice, will also help getting the `Compute/FormComputeDataset.tsx` smaller in https://github.com/oceanprotocol/market/pull/439

2 small things:
1. naming: always hard, and I can see how `getAlgorithms()` is too generic since they are returned in a specific way which `...Options` tries to convey but it's also very generic. Maybe we should be precise with `getAlgorithmsForAssetSelection()` or `getAlgorithmsForSelection()` or something else to signal what is returned, or the use case
2. config typing: we can type the `config` parameter, usually `ConfigHelperConfig` | `Config` works